tests/receivers/discovery/testdata/host_observer_simple_prometheus_config.yaml

```diff
@@ -13,14 +13,14 @@ receivers:
     embed_receiver_config: true
     receivers:
       prometheus_simple:
-        rule: type == "hostport"
+        rule: type == "hostport" and command contains "otelcol"
```

Comment:
Without this change in rule, too many ports were being discovered and the test would timeout before it even tried the internal metrics port. I also updated the metric to match against for a successful status to get more reliable runs.